frontend/layout/navbar/component.tsx

```diff
@@ -1,30 +1,48 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
+import cx from 'classnames';
 
 import Link from 'next/link';
 import Icon from 'components/icon';
+import Hamburger from 'components/hamburger';
 import FacebookIcon from '../../svgs/social/facebook.svg'; // better to use something else, inline react does not work well with typescript, webpack5
```

Comment:
I think we could use svgr and import it as react components. See example: https://dev.to/dolearning/importing-svgs-to-next-js-nna

Reply:
noted. Or IconMoon with no webpack plugins

Reply:
Sure, the standard that we use now is svgo instead of svgr. I was not sure